aten/src/ATen/native/native_functions.yaml

```diff
@@ -1226,6 +1226,34 @@
   use_c10_dispatcher: full
   variants: method
 
+  # true_divide, an alias for div
+- func: true_divide.Tensor(Tensor self, Tensor other) -> Tensor
+  use_c10_dispatcher: full
+  variants: function, method
+  dispatch:
```

Comment:
Does setting dispatch override the gradient? Check what test would catch this.

Delete dispatches.

Reply:
Interesting follow-up: probably want to add gradcheck to test_op_aliases, method_test entries missing for true_divide and floor_divide.

Reply:
Explicitly dispatching was causing autograd to fail. Deleting the dispatches works. Unfortunately aliases can't be added to method_tests() without additional test changes because test_jit.py has a check where it looks for the node, by name, but the node doesn't appear in the JIT's graph (because it's been aliased to another op). In the near future aliases will be refactored into OpInfos and gradcheck'd automatically there.